digiroad2-oracle/src/test/resources/fi/liikennevirasto/digiroad2/util/testChangeSets/add.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+  "changeType": "add",
+  "old": [],
+  "new": [
+    {
+      "linkId": "newLink:1",
+      "length": 100.0,
+      "geometry":  "LINESTRING ZM()",
+      "roadClass": 12121,
+      "adminClass": 2,
+      "municipality": 235,
+      "trafficDirection": 0
+    }
+  ],
+  "replaceInfo": []
+  }
+]
```

Comment:
Laittaisin mieluummin kaikki tapaukset yhteen samaan testi tiedostoon. S3:een tallennettava tiedosto tulee kuitenkin pitämään sisällään useita muutoksia.

Oma ajatus oli, että testitiedostoon sisällytettäisiin tosielämän muutoksia oikeilla linkeillä (näitä löytyy siitä Sasun koostamasta tiedostosta Teamsista), jotka ideaalissa tilanteessa olisi kaikki saman kunnan alueelta, mikä helpottaisi testaamista. Samuutuksessa linkin geometria on oleellinen tieto, eikä se voi olla tyhjä.